ow/launch/common.launch

```diff
@@ -74,6 +74,9 @@
 
   <!-- Start faults node -->
   <node name="faults" pkg="ow_faults" type="faults" output="screen" />
+  
+  <!-- Start faults detection node -->
+  <node name="faults_detection" pkg="ow_faults_detection" type="faults_detection" output="screen" />
```

Comment:
Low priority now, but later, would like these packages renamed in a more clear and consistent way, e.g. "fault_injection" and "fault_detection".

Reply:
done and updated a PR for autonomy due to this name change.